src/web/dataadapters/GreyNoiseAdapterSummary.jsx

```diff
@@ -31,13 +33,17 @@ class GreyNoiseAdapterSummary extends React.Component {
   };
 
   render() {
-    const { config } = this.props.dataAdapter;
-
     return (
-      <dl>
-        <dt>API Token</dt>
-        <dd>******</dd>
-      </dl>
+        <div>
+          <dl>
+            <dt>API Token</dt>
+            <dd>******</dd>
+          </dl>
+          <Alert style={{marginBottom: 10}} bsStyle="warning">
```

Comment:
```suggestion
          <Alert style={{marginBottom: 10}} bsStyle="danger">
```
Very small nit, but what do you think about showing this in red instead? Just thinking that since the adapter can no longer be used, perhaps red might make sense.

Reply:
Good call the scarier the better :grin:, pushing an update. Thanks for the review and testing!